### ISSUE TITLE
[TC-PRS-2.1] MaxMeasuredValue attribute value mismatches from specification.

### ISSUE THREAD

**Issue by Rajashreekalmane** [project-chip/connectedhomeip#27363]
### Reproduction steps

1. executed the below mentioned command to put DUT into a commissionable state
` ./chip-all-clusters-app
`
2. Once DUT reach the  commissionable state sent below mentioned command on TH. 
` ./chip-tool pairing onnetwork 1 20202021`
Verify the commissioning completed with success on TH(chip-tool) from DUT
3. TH reads from the DUT the MinMeasuredValue attribute : `./chip-tool pressuremeasurement read min-measured-value 1 1`
4. TH reads from the DUT the MaxMeasuredValue attribute : `./chip-tool pressuremeasurement read max-measured-value  1 1`

### Bug prevalence

Everytime

### GitHub hash of the SDK that was being used

Master - 40fb7c246671e9d37ca8bb2cfbf37f81c1509334	

### Platform

raspi

### Platform Version(s)

RPI4 8GB RAM

### Anything else?

As per Spec MaxMeasuredValue Attribute indicates the maximum value of MeasuredValue that can be measured and the value should be MinMeasuredValue+1 to 32767. But In master commit while TH trying to read the  MinMeasuredValue and MaxMeasuredValue attribute value reads  0 from DUT for both attributes. 

**PFA Spec screenshot** 
![image](https://github.com/project-chip/connectedhomeip/assets/95841834/380e7d81-d845-4cad-a09b-b16c4b54d8d8)

./chip-tool pressuremeasurement read min-measured-value 1 1
```
[1687328121.962035][142811:142813] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0403 Attribute 0x0000_0001 DataVersion: 3881742437
[1687328121.962092][142811:142813] CHIP:TOO:   MinMeasuredValue: 0
[1687328121.962179][142811:142813] CHIP:EM: <<< [E:62637i S:43113 M:263665961 (Ack:56316814)] (S) Msg TX to 
```

./chip-tool pressuremeasurement read max-measured-value  1 1
```
[1687328180.923268][142828:142830] CHIP:TOO: Endpoint: 1 Cluster: 0x0000_0403 Attribute 0x0000_0002 DataVersion: 3881742437
[1687328180.923326][142828:142830] CHIP:TOO:   MaxMeasuredValue: 0
[1687328180.923414][142828:142830] CHIP:EM: <<< [E:9978i S:44691 M:127860509 (Ack:224292634)] (S) Msg TX to 
```

**PFA reference log :** 
[PRS-2_1.txt](https://github.com/project-chip/connectedhomeip/files/11814175/PRS-2_1.txt)

**Comment by bzbarsky-apple** [comment]
Yep, the ZAP config for all-clusters-app does not list any actual values for Min/MaxMeasuredValue, so they end up defaulting to 0.
